portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase

```diff
@@ -325,11 +210,229 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for OAUTH2-259. Assert client credential is user scoped."
+	@description = "This is a use case for LPS-129397 TC-1: Revoke multiple authorization via new kebab menu option in OAuth2 application's row."
+	@priority = "5"
+	test AuthorizationCanBeRevokedViaKebabMenu {
+		property test.name.skip.portal.instance = "OAuth2#AuthorizationCanBeRevokedViaKebabMenu";
+
+		task ("Add and edit an external application") {
+			OAuth2.addAndEditExternalApplication(
+				applicationName = "My application",
+				callbackURI = '''http://martamedio.com/oauth2-tester/?url=${portalURL}&client_id=default''',
+				clientId = "default",
+				clientProfile = "Web Application",
+				clientSecret = "defaultsecret",
+				websiteURL = "http://localhost:8080");
+		}
+
+		task ("Assert application name is the index 0 at OAuth2Admin") {
+			OAuth2.openOAuth2Admin();
+
+			OAuth2.viewApplicationInAuthorizationsTable(
+				entryIndex = "0",
+				tableEntry = "My application");
+
+			LexiconEntry.viewNoEntryMenuItem(
+				menuItem = "Revoke Authorizations",
+				rowEntry = "My application");
+		}
+
+		task ("Authorize application and launch request") {
+
+			// Authorize the application
```

Comment:
I think the task group actually replaces the comments we use, or not?
@ferenconodiD